docs/core/event_handler/bedrock_agents.md

```diff
@@ -0,0 +1,330 @@
+---
+title: Agents for Amazon Bedrock
+description: Core utility
+---
+
+Author [Agents for Amazon Bedrock](https://docs.aws.amazon.com/bedrock/latest/userguide/agents.html#agents-how){target="_blank"} using event handlers and auto generation of OpenAPI schemas.
+
+<center>
+```mermaid
+--8<-- "docs/core/event_handler/bedrock_agents.mermaid"
+```
+</center>
+
+## Key features
+
+* Similar experience when authoring [REST](api_gateway.md){target="_blank"} and [GraphQL APIs](appsync.md){target="_blank"}
+* Minimal boilerplate to build Agents for Amazon Bedrock
+* Automatic generation of [OpenAPI schemas](https://www.openapis.org/){target="_blank"} from your business logic code
+* Built-in data validation for requests and responses
+
+## Terminology
+
+**Data validation** automatically validates the user input and the response of your AWS Lambda function against a set of constraints defined by you.
+
+**Event handler** is a Powertools for AWS feature that processes an event, runs data parsing and validation, routes the request to a specific function, and returns a response to the caller in the proper format.
+
+**[OpenAPI schema](https://www.openapis.org/){target="_blank"}** is an industry standard JSON-serialized string that represents the structure and parameters of your API.
+
+**Action group** is a collection of two resources where you define the actions that the agent should carry out: an OpenAPI schema to define the APIs that the agent can invoke to carry out its tasks, and a Lambda function to execute those actions.
+
+**Large Language Models (LLM)** are very large deep learning models that are pre-trained on vast amounts of data, capable of extracting meanings from a sequence of text and understanding the relationship between words and phrases on it.
+
+**Agent for Amazon Bedrock** is an AWS service to build and deploy conversational agents that can interact with your customers using Large Language Models (LLM) and AWS Lambda functions.
```

Comment:
this is semantically incorrect, AAB is a feature of bedrock rather than a service.